sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/models.py

```diff
@@ -515,11 +515,24 @@ def request_id(self):
 class CertificatePolicy(object):
     """Management policy for a certificate.
 
-    :param attributes: the certificate attributes.
+    :param attributes: The certificate attributes. These are set by the service.
     :type attributes: ~azure.keyvault.certificates.models.CertificateAttributes
     :param str cert_policy_id: The certificate id.
-    :param key_properties: Properties of the key backing the certificate.
-    :type key_properties: ~azure.keyvault.certificates.models.KeyProperties
+    :param bool exportable: Indicates if the private key can be exported.
+    :param key_type: The type of key pair to be used for the certificate.
+        Possible values include: 'EC', 'EC-HSM', 'RSA', 'RSA-HSM', 'oct'
```

Comment:
Could just reference the `KeyType` enum here.

Reply:
got rid of that reference, thanks!

Reply:
got rid of links to enums in comments, thanks!